app/utilities/schema.py

```diff
@@ -80,9 +86,12 @@ def load_schema_from_session_data(session_data):
     return load_schema_from_metadata(vars(session_data))
 
 
+def load_schema_from_name(schema_name, language_code=DEFAULT_LANGUAGE_CODE):
+    return _load_schema_from_name(schema_name, language_code)
```

Comment:
Is there any value in this? Could we not just drop the `_` from `_load_schema_from_name` and call it directly?

Reply:
It was so that `load_schema_from_name` can be used with args or keyword args and still fetch from cache, as the cache depends on the method signature. If we are happy with, the rule that you must not use keyword args or must match whatever the format for loading the cache is then ya I could remove.